code/modules/events/spacevine.dm

```diff
@@ -466,9 +466,10 @@
 		return
 	if(parent)
 		SV.mutations |= parent.mutations
-		var/parentcolor = parent.atom_colours[FIXED_COLOUR_PRIORITY]
-		SV.add_atom_colour(parentcolor, FIXED_COLOUR_PRIORITY)
-		if(prob(mutativness))
+		if(parent.atom_colours)
```

Comment:
in fact, it's the controller that isn't calling ..() in New().

So this list is never init'ed.